app.js

```diff
@@ -1,73 +1,82 @@
+new Product('tauntaun', 'img/tauntaun.jpg', 'tauntaun');
+new Product('unicorn', 'img/unicorn.jpg', 'unicorn');
+new Product('usb', 'img/usb.gif', 'usb');
+new Product('water can', 'img/water-can.jpg', 'water can');
+new Product('wine glass', 'img/wine-glass.jpg', 'wine glass');
 
 var numbOne;
 var numbTwo;
 var numbThree;
 var oldArray = [];
 
-var randomIndex = function(){
+function randomProduct(){
   numbOne = (Math.floor(Math.random() * Product.allProducts.length));
   numbTwo = (Math.floor(Math.random() * Product.allProducts.length));
   numbThree = (Math.floor(Math.random() * Product.allProducts.length));
-};
-
-function randomProduct(){
-  randomIndex();
   while ((numbOne === numbTwo) || (numbOne === numbThree) || (numbTwo === numbThree) || ((oldArray.indexOf(numbOne) !== -1) || (oldArray.indexOf(numbTwo) !== -1) || (oldArray.indexOf(numbThree) !== -1))) {
-    randomIndex();
+    numbOne = (Math.floor(Math.random() * Product.allProducts.length));
+    numbTwo = (Math.floor(Math.random() * Product.allProducts.length));
+    numbThree = (Math.floor(Math.random() * Product.allProducts.length));
   }
   oldArray = [];
-  var imgEl = document.getElementById('render1');
-  imgEl.addEventListener('click', randomProduct);
-  // var randomIndex = Math.floor(Math.random() * Product.allProducts.length);
-  imgEl.src = Product.allProducts[numbOne].filepath;
-  Product.allProducts[numbOne].clickCounter += 1;
-  var imgEl2 = document.getElementById('render2');
-  imgEl2.addEventListener('click', randomProduct);
-  imgEl2.src = Product.allProducts[numbTwo].filepath;
-  Product.allProducts[numbTwo].clickCounter += 1;
-  var imgEl3 = document.getElementById('render3');
-  imgEl3.addEventListener('click', randomProduct);
-  imgEl3.src = Product.allProducts[numbThree].filepath;
-  Product.allProducts[numbThree].clickCounter += 1;
   oldArray.push(numbOne);
   oldArray.push(numbTwo);
   oldArray.push(numbThree);
-}
+  var imgEl = document.getElementById('click1');
+  imgEl.src = Product.allProducts[numbOne].filepath;
+  Product.allProducts[numbOne].timesRendered++;
+  var imgEl2 = document.getElementById('click2');
+  imgEl2.src = Product.allProducts[numbTwo].filepath;
+  Product.allProducts[numbTwo].timesRendered++;
+  var imgEl3 = document.getElementById('click3');
+  imgEl3.src = Product.allProducts[numbThree].filepath;
+  Product.allProducts[numbThree].timesRendered++;
+};
 
-// next steps: check that randomProduct() only loops x25, then create new function that uses innerHTML to blank out images and then replaces that content with new getElementById on an <ul> that appends .clickCounter and iterates through Product.allProducts[x].clickCounter!
-var i = 0;
-while (i < 25) {
-  randomProduct();
-  i++;
-}
+function clickChoice(e) {
+  if(e.target.id === 'images') {
+    return alert('Please click on an image itself');
+  }
+  for(var i = 0; i < Product.allProducts.length; i++) {
+    if(event.target.alt === Product.allProducts[i].alt) {
```

Comment:
This event does not trigger for a couple reasons.

Here on the left side of the comparison you are referencing event.target.alt, but event is not the name of the argument you are passing. Also, if you should log both sides of this if statement separately to see what properties they have, and what is being compared.